edweiss-app/model/time.ts

```diff
@@ -3,7 +3,7 @@ export type WeekDay = number & {};
 
 export type Minutes = number & {};
 
-export interface Timestamp {
-	readonly seconds: number;
-	readonly nanoseconds: number;
-}
+/**
```

Comment:
This comments are useless, you can add this code or just delete him.

Reply:
Actually, this is a important point to highlight. This is the contributes to the date fix that I mentionned earlier. I am exposing only the interface so that it's compatible with the FB Timestamp to ensure reusabilty and also ease of conversion between it and JS Date.